pager/pbar.c

```diff
@@ -110,7 +110,8 @@ static int pbar_recalc(struct MuttWindow *win)
 
   if (offset < (priv->st.st_size - 1))
   {
-    snprintf(pager_progress_str, sizeof(pager_progress_str), OFF_T_FMT "%%",
+    /* L10N: Localized percentage formatting */
+    snprintf(pager_progress_str, sizeof(pager_progress_str), _("%s%%"),
```

Comment:
This is kind of a blind fix :D.

Reply:
er, ok.
that's a little trickier than I thought.
I'll see what I can do about refactoring it.

Reply:
Shall I just pull this back for the moment? I'll fix the comments for other parts.

Reply:
Fixed. I hope I got the context right. I amended them into your commit by mistake, let me know if you prefer another commit.